src/main/resources/assets/create/lang/zh_cn.json

```
@@ -293,6 +302,8 @@
 	"block.create.mechanical_pump": "动力泵",
 	"block.create.mechanical_saw": "动力锯",
 	"block.create.metal_bracket": "金属支架",
+	"block.create.metal_girder": "金属梁",
+	"block.create.metal_girder_encased_shaft": "金属梁传动杆箱",
```

```suggestion
	"block.create.metal_girder_encased_shaft": "传动金属梁",
```
这样会不会更好？

---

金属梁包裹传动杆

---

恭喜你发现了这个远古问题（
你可以看看别的装壳xxx，应该都有类似的翻译成xx箱

---

别的翻译成箱没啥问题，看起来确实是箱
但这个，怎么看也不像“箱”

---

是的，所以说是上古大坑，我只是遵从统一暂时翻译成箱了。你看看别的能不能也帮忙修正一下

---

原文

> Metal Girder Encased Shaft

翻译成“传动金属梁”不太好，因为主语是“传动杆”，所以还是翻译成这样吧

```suggestion
	"block.create.metal_girder_encased_shaft": "金属梁包住的传动杆",
```